auth_service/src/AuthService/AuthService/views.py

```diff
@@ -0,0 +1,108 @@
+import json
+
+from tornado.web import RequestHandler
+from authlib.jose import jwt, JsonWebKey
+from cryptography.hazmat.primitives.serialization import load_pem_private_key
+from cryptography.hazmat.backends import default_backend
+from AuthService import settings
+
+DEMO_PRIVATE_KEY = b"""-----BEGIN RSA PRIVATE KEY-----
```

Comment:
@newgene Note: this is just a demo key that isn't used anywhere else, so there's not a security issue with having it in the repo. I will be replacing this with an environment variable in the next round of improvements to this code